README.md

```diff
@@ -56,7 +56,7 @@ Many board that run [Armbian](https://www.armbian.com/) such as many models of O
 | `WITH_FIREWALL`                 | None                                           | Set to `true` if you want to enable a basic firewall on your node.|
 | `WITH_CJDNS_IPTUNNEL`           | None                                           | Set to `true` if you want to use the cjdns iptunnel feature to set up an Internet gateway for your node. To configure as a server (exit Internet traffic for other nodes), create **/etc/cjdns.iptunnel.server** containing a newline-separated list of cjdns public keys of allowed clients. To configure as a client (use an exit server to access the Internet), create **/etc/cjdns.iptunnel.client** containing a newline-separated list of cjdns public keys of the gateway servers. You can only configure as one or the other, not both. |
 | `WITH_IPFS`                     | **80**: HTTP-to-IPFS gateway at `/ipfs/HASH`   | Set to `true` if you want to install [IPFS](https://ipfs.io). |
-| `WITH_IPFS_PI_STREAM`           | None                                           | Set to `true` if you want to install Pi stream service to live stream your camera over IPFS. Requires a Raspberry Pi with camera module. *Will automatically start recording on boot by default.* |
+| `WITH_IPFS_PI_STREAM`           | None                                           | Set to `true` if you want to install Pi stream service to live stream your camera over IPFS. Requires a Raspberry Pi with camera module.  Player interface at `/video-player` *Will automatically start recording on boot by default.* |
```

Comment:
The sentence now reads very awkward... I think you mean...

>Automatically start recording on boot, playback available at `/video-player`.

One space after periods like everywhere else in this repo.

Reply:
re-worded as

> Set to true if you want to install Pi stream service to live stream your camera over IPFS. Requires a Raspberry Pi with camera module. Player interface at /video-player Warning: By default the camera will start broadcasting after first reboot.